devicesetup/build.gradle

```diff
@@ -1,7 +1,9 @@
 apply plugin: 'com.android.library'
+apply plugin: 'maven'
+group = 'com.github.Sushinn'
```

Comment:
This is used to import the library on gradle with `jitpack.io`. It was added so I could use the lib on the project that I'm working. I'll remove this for the PR